build.gradle

```diff
@@ -23,11 +23,16 @@ repositories {
     mavenCentral()
     maven { url "https://hub.spigotmc.org/nexus/content/repositories/snapshots/" }
     maven { url "https://oss.sonatype.org/content/repositories/snapshots" }
+    maven {
+        name 'velocity'
```

Comment:
I don't think there's a need to include the repo name - we're not doing any complex module stuff, and would prefer it to in the same format as the other repos

Reply:
Changing that as soon I'm on my coding PC